src/libsyncengine/syncpal/tmpblacklistmanager.h

```diff
@@ -35,14 +35,15 @@ class TmpBlacklistManager {
         TmpBlacklistManager(std::shared_ptr<SyncPal> syncPal);
         ~TmpBlacklistManager();
 
-        void increaseErrorCount(const NodeId &nodeId, NodeType type, const SyncPath &relativePath, ReplicaSide side);
-        void blacklistItem(const NodeId &nodeId, const SyncPath &relativePath, ReplicaSide side);
+        void increaseErrorCount(const NodeId &nodeId, NodeType type, const SyncPath &relativePath, const ReplicaSide side);
+        void blacklistItem(const NodeId &nodeId, const SyncPath &relativePath, const ReplicaSide side);
         void refreshBlacklist();
-        void removeItemFromTmpBlacklist(const NodeId &nodeId, ReplicaSide side);
-        bool isTmpBlacklisted(ReplicaSide side, const SyncPath &path);
+        void removeItemFromTmpBlacklist(const NodeId &nodeId, const ReplicaSide side);
```

Comment:
Do you think `const` is important here?

Reply:
Not important at all. Just wanted to make it more explicit that this is an input argument.
But you are right, there is actually no reason to pass it as const according to [c++ guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rf-in).
I will remove that